src/dbnode/storage/index.go

```diff
@@ -861,17 +864,16 @@ func (i *nsIndex) BootstrapsDone() uint {
 }
 
 func (i *nsIndex) Tick(c context.Cancellable, startTime time.Time) (namespaceIndexTickResult, error) {
-	var (
-		result                     = namespaceIndexTickResult{}
-		earliestBlockStartToRetain = retention.FlushTimeStartForRetentionPeriod(i.retentionPeriod, i.blockSize, startTime)
-	)
+	var result = namespaceIndexTickResult{}
```

Comment:
This can just be `var result namespaceIndexTickResult` which is simpler than creating a concrete initialization.

If you wanted to go the later then it would be better to `result := namespaceIndexTickResult{}`

Reply:
👍